rClr.props

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ImportGroup Label="PropertySheets" />
+  <PropertyGroup Label="UserMacros">
+    <RInstallPath>C:\PROGRA~1\R\R-43~1.1</RInstallPath>
```

Comment:
I had to hard-code the path to R installation here, this should be done automatically somewhow.